packages/lit-dev-content/src/components/litdev-aside.ts

```diff
@@ -43,13 +43,17 @@ export class LitDevAside extends LitElement {
       font-weight: bold;
     }
 
-    ::slotted(:first-of-type),
-    ::slotted(:nth-of-type(2)) {
+    :host(:not([no-header])) ::slotted(:first-child),
+    :host(:not([no-header])) ::slotted(:nth-child(2)) {
+      display: inline;
+    }
+
+    ::slotted(:first-child) {
       margin-block-start: 0;
     }
 
-    ::slotted(:first-of-type),
-    ::slotted(:last-of-type) {
+    ::slotted(:first-child),
```

Comment:
this was a bug. it would bold both these:

```
<litdev-aside>

bolded first line

`bolded second line`

</litdev-aside>
```
because first is a `<p>` second is a `<code>` and they are each first-of-type. oops!